### ISSUE TITLE
docker_registry.sh incorrect causing registry not to start?

### ISSUE THREAD

**Issue by SpencerBrown** [TheNewNormal/coreos-osx#67]
```

start() {
    echo "Starting Docker Registry v2 on 192.168.64.1:5000 ..."
    nohup registry config.yml >/dev/null 2>&1 &
}

stop() {
    kill $(ps aux | grep "[r]egistry config.yml" | awk {'print $2'}) >/dev/null 2>&1
```

I believe there needs to be a `serve` subcommand after `registry` or it won't work.

At least that's my experience trying 1.2.9.

**Comment by rimusz** [comment]
@SpencerBrown thanks, good catch, as `registry` got that `serve` recently

**Comment by rimusz** [comment]
issue fixed with v1.3.0 #68
